<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverlayAnalysisWindowClass</class>
 <widget class="QDialog" name="OverlayAnalysisWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>445</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>445</width>
    <height>445</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>445</width>
    <height>445</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>叠加分析</string>
  </property>
  <widget class="QPushButton" name="startpushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelpushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>相交图层：</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="9" column="1">
     <widget class="QToolButton" name="toolButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>输出要素：</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>连接属性：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="analysecomboBox"/>
    </item>
    <item row="7" column="0">
     <widget class="QComboBox" name="AttributecomboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>输入图层：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="inputcomboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
